pkg/leaderelection/config.go

```diff
@@ -27,9 +27,11 @@ import (
 var (
 	validComponents = sets.NewString(
 		"controller",
+		"contour-ingress-controller",
 		"hpaautoscaler",
 		"certcontroller",
 		"istiocontroller",
+		"net-http01",
```

Comment:
Would this be pushed to their own repos at some point?

Reply:
Given how we've centralized this configmap validation it's not really plausible right now.  However, I'd like to move to enable this by default and eliminate this flag ASAP in 0.17.